text/src/autogluon/text/automm/utils.py

```diff
@@ -450,16 +450,18 @@ def create_model(
     # make sure no duplicate model names
     assert len(names) == len(set(names))
     logger.debug(f"output_shape: {num_classes}")
+    names = [n.lower() for n in names]
     all_models = []
     for model_name in names:
-        model_config = getattr(config.model, model_name)
-        if model_name == "clip":
```

Comment:
We may not use “in”.

Reply:
Per offline discussion, we may use "startswith" and add a new test-case about having two timm models.